clients/desktop/src/mpc/state/vaultCreationMpcLib.ts

```
@@ -1,9 +1,7 @@
 import { defaultMpcLib, MpcLib } from '@core/mpc/mpcLib'
 
-import {
-  PersistentStateKey,
-  usePersistentState,
-} from '../../state/persistentState'
+import { usePersistentState } from '../../state/persistentState'
+import { PersistentStateKey } from '@core/ui/state/PersistentStateKey'
```

ESLint

[error] Run autofix to sort these imports!

(simple-import-sort/imports)